view/frontend/web/js/pargo.js

```diff
@@ -8,668 +8,626 @@
+                observer.observe(doc.documentElement, {
+                    childList: true,
+                    subtree: true,
+                });
+            }
+            // Check if the element is currently in the DOM
+            check();
+        }
 
-    function check() {
-      // Check the DOM for elements matching a stored selector
-      for (
-        var i = 0, len = listeners.length, listener, elements;
-        i < len;
-        i++
-      ) {
-        listener = listeners[i];
-        // Query for elements matching the specified selector
-        elements = doc.querySelectorAll(listener.selector);
-        for (var j = 0, jLen = elements.length, element; j < jLen; j++) {
-          element = elements[j];
-          // Make sure the callback isn't invoked with the
-          // same element more than once
-          if (!element.ready) {
-            element.ready = true;
-            // Invoke the callback with the element
-            listener.fn.call(element, element);
-          }
+        function check() {
+            // Check the DOM for elements matching a stored selector
+            for (
+                let i = 0, len = listeners.length, listener, elements;
+                i < len;
+                i++
+            ) {
+                listener = listeners[i];
+                // Query for elements matching the specified selector
+                elements = doc.querySelectorAll(listener.selector);
+                for (let j = 0, jLen = elements.length, element; j < jLen; j++) {
+                    element = elements[j];
+                    // Make sure the callback isn't invoked with the
+                    // same element more than once
+                    if (!element.ready) {
+                        element.ready = true;
+                        // Invoke the callback with the element
+                        listener.fn.call(element, element);
+                    }
+                }
+            }
         }
-      }
-    }
 
-    // Expose `ready`
-    win.ready = ready;
-  })(this);
-
-  ready(".opc-progress-bar", function () {
-    $(".opc-progress-bar").on('click', function () {
-      if (
-        $("input[value='pargo_customshipping_pargo_customshipping']").prop(
-          "checked"
-        ) &&
-        localStorage.getItem("pargoPoint") != null
-      ) {
-        //$('.form-shipping-address').hide();
-        //$('.checkout-shipping-address').hide();
-        $(".continue").attr("disabled", false);
-      }
+        // Expose `ready`
+        win.ready = ready;
+    })(this);
+
+    ready(".opc-progress-bar", function () {
+        $(".opc-progress-bar").on('click', function () {
+            if (
+                $("input[value='pargo_customshipping_pargo_customshipping']").prop(
+                    "checked"
+                ) &&
+                localStorage.getItem("pargoPoint") !== null
+            ) {
+                //$('.form-shipping-address').hide();
```

Comment:
Remove commented code.

Reply:
This is my second code clean up of this file, I'm still working out why some of the code is commented out in the first place. Will continue with the clean up in the next iteration, right now I need to see if this fixes bugs experienced.